java/src/jmri/jmrit/beantable/BeanTableDataModel.java

```diff
@@ -43,14 +43,15 @@
 import javax.swing.table.TableCellEditor;
 import javax.swing.table.TableColumn;
 import javax.swing.table.TableModel;
+import javax.swing.table.TableRowSorter;
 import jmri.InstanceManager;
 import jmri.JmriException;
 import jmri.Manager;
 import jmri.NamedBean;
 import jmri.NamedBeanHandleManager;
 import jmri.UserPreferencesManager;
 import jmri.swing.JTablePersistenceManager;
-import jmri.util.com.sun.TableSorter;
+import jmri.util.com.sun.TableSorter; // still used?
```

Comment:
This question is *very* easily answered by looking at the file that imports it. It's still used, as a quick scan of this file would reveal. It is deprecated, but that does not mean we get to remove it immediately, because unless the use of something is in a private field or method, there is no reasonable expectation that it's not been used in some third party software that is built on top of JMRI.